### ISSUE TITLE
IQApplication connect times out 75% of the time

### ISSUE THREAD

**Issue by QueenBee61** [QuickbaseAdmirer/QuickBase-C-Sharp-SDK#9]
This is a real issue for us and our client.  I have the following code:
```
      IQClient client = QuickBase.Login(userName, password, "mydatabase.quickbase.com");
            IQApplication app = client.Connect("bnuy96t9h", "d9rdib9g2ju4fbqg68hccuxraw6");
```
when it reaches "(IQApplication app = client.Connect)"  it times out 75% of the time, and if it doesn't time out, it takes about 5-8 minutes to get past.  I do not know what to do about it.  And either us , Quickbase, or both are going to lose a client over it.

**Comment by jherig** [comment]
Question: do you have a lot of tables in your app, and/or a huge number of fields per table?  There is a very large overhead cost in creating the IQApplication object.  One of the issues on my plate is to make this 'lazy loading' which would help if that's the problem.  I can try to prioritize that, if this seems to describe your issue.

**Comment by QueenBee61** [comment]
Yep, that app has a lot of tables, and I don't know about all of the tables, but the one I am accessing has 89 fields.   Is there another way to get the records without creating that object?  I am trying to get all records from this one particular table, and am trying to add records to that same table.  The other tables do not concern me at all.

**Comment by jherig** [comment]
No, there is currently no way to manipulate a table without creating the IQApplication itself.  Let me see how much work it would be to lazy load the tables and put that into app.GetTable instead.
